internal/controller/imageupdateautomation_controller.go

```diff
@@ -92,7 +92,10 @@ func getPatchOptions(ownedConditions []string, controllerName string) []patch.Op
 
 // +kubebuilder:rbac:groups=image.toolkit.fluxcd.io,resources=imageupdateautomations,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=image.toolkit.fluxcd.io,resources=imageupdateautomations/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=image.toolkit.fluxcd.io,resources=imagepolicies,verbs=get;list;watch
+// +kubebuilder:rbac:groups=image.toolkit.fluxcd.io,resources=imagepolicies/status,verbs=get
 // +kubebuilder:rbac:groups=source.toolkit.fluxcd.io,resources=gitrepositories,verbs=get;list;watch
+// +kubebuilder:rbac:groups="",resources=events,verbs=create;patch
```

Comment:
It also needs `get;list;watch` for Kubernetes Secrets to read the Git auth.

Reply:
I don't see Kubernetes secret kind added to this list

Reply:
I misunderstood the comment earlier, fixed now. Thanks.